test/storage.jl

```diff
@@ -145,5 +216,35 @@ end
   @test g2.attrs == Dict("groupatt"=>5)
   @test g2["a1"].attrs == Dict("arratt"=>2.5)
   @test g2["a1"][:,:] == reshape(1:200,10,20)
+  
+  # The following test doesn't pass, but maybe should?
```

Comment:
Ok, I will check this later. Maybe we could make a broken test from it, let's see